### DISCUSSION TITLE
ConnectionError: HTTPSConnectionPool(host='coqui.gateway.scarf.sh', port=443): Max retries exceeded with url: /trainer/training_run

### DISCUSSION THREAD

**Discussion by naveenkbc** [coqui-ai/TTS#2344] [General Q&A]
I am trying to train a model from previous checkpoint. When program invokes trainer.fit() method, its is trying to connect coqui.gateway.scarf.sh host system with url /trainer/training_run. Is there a way to disable this option as I have everything in local environment ?

**Comment by erogol** [comment]
https://github.com/coqui-ai/Trainer#anonymized-telemetry
